wordpress-mobile/gutenberg-mobile
# Enable page templates

This PR removes the `__mobileEnablePageTemplates` feature flags and enables page templates by default.

Fixes #1962 

gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1987
gutenberg PR: https://github.com/WordPress/gutenberg/pull/20718
WPiOS PR: https://github.com/wordpress-mobile/WordPress-iOS/pull/13595
WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/11409

To test:

From either WPiOS or WPAndroid, create a new page and verify that the template picker is showing

PR submission checklist:
